src/ophys_etl/modules/segmentation/modules/schemas.py

```diff
@@ -222,3 +222,43 @@ class SimpleDenoiseInputSchema(argschema.ArgSchema, DenoiseBaseSchema):
         default=1,
         description=("how many multiprocessing workers to use. If set to "
                      "1, multiprocessing is not invoked."))
+
+
+class FeatureVectorSegmentationInputSchema(argschema.ArgSchema):
+    log_level = argschema.fields.LogLevel(default="INFO")
+
+    video_input = argschema.fields.InputFile(
+        required=False,
+        description=("path to hdf5 video with movie stored "
+                     "in dataset 'data' nframes x nrow x ncol"))
+
+    graph_input = argschema.fields.InputFile(
+        required=False,
+        description=("path to graph used to seed ROIS"))
+
+    attribute = argschema.fields.Str(
+        required=False,
+        default="Pearson",
```

Comment:
I'm guessing `filtered_hnc_Gaussian` will be the go-to, should we just change this?

Reply:
I also wonder if we should just have an `allow_none` and in a marshmallow post-load, if it is None, look to see if there is one and only one edge attribute, and then assign that, with a message.

Reply:
don't need for development